test/parallel/test-net-write-after-close.js

```diff
@@ -44,7 +42,10 @@ server.listen(0, function() {
     // Then 'end' will be emitted when it receives a FIN packet from
     // the other side.
     client.on('end', common.mustCall(() => {
-      serverSocket.write('test', common.mustCall());
+      serverSocket.write('test', common.mustCall((err) => {
+        assert(err);
+        server.close();
+      }));
```

Comment:
`serverSocket` has already been destroyed to further errors won't be propagated through `'error'`.